Common/Gfx/D2DBitmap.cpp

```diff
@@ -51,6 +51,10 @@ D2DBitmap::D2DBitmap(const std::wstring& path, int exifOrientation) :
 {
 }
 
+D2DBitmap::D2DBitmap(void* ptr) : D2DBitmap()

```

Comment:
I could just make the default constructor public but i'm afraid that i break something.

Reply:
I don't think it will break anything.